detector/response_cache.go

```diff
@@ -0,0 +1,11 @@
+package detector
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/iam"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/aws/aws-sdk-go/service/iam (can't find import: github.com/wata727/tflint/vendor/github.com/aws/aws-sdk-go/service/iam)

<!-- Comment by [SideCI](https://sideci.com) -->